target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -18,6 +18,7 @@ board_config_update
 
 case $board in
 11acnas|\
+afoundry-ew1200|\
```

Comment:
Actually, the full name of "ZBT" company is (based on their website) "Shenzhen Zhibotong Electronics Co., Ltd" but more often used name is probably "ZBTlink" or just "ZBT". What is important here, all ZBT devices are named "ZBT-something" by the manufacturer, we didn't add "ZBT" as a company prefix in the code.